components/automate-ui/src/app/modules/infra-proxy/data-bags-details/data-bags-details.component.ts

```diff
@@ -205,7 +205,7 @@ export class DataBagsDetailsComponent implements OnInit, OnDestroy {
 
   public startUpdateDataBagItem(item: DataBagItems, jsonData: Object): void {
     this.dataBagItemName = item.name;
-    this.itemDataJson = JSON.stringify(jsonData, undefined, 4);
+    this.itemDataJson = JSON.stringify(jsonData, null, 4);
```

Comment:
Author note: wrong values - the behavior when using undefined is, well, undefined 🤷 ! The correct usage is with null.